compiler/rustc_lint/src/internal.rs

```diff
@@ -285,6 +302,8 @@ impl EarlyLintPass for LintPassImpl {
 }
 
 declare_tool_lint! {
+    /// The `existing_doc_keyword` lint detects use `#[doc()]` keywords
+    /// that don't exist, e.g. `#[doc(keyword = "..")]`.
     pub rustc::EXISTING_DOC_KEYWORD,
```

Comment:
i dont quite understand why this lint exists in the first place, it doesn't seem too useful? #79541 doesn't contain any info
but whatever, not that it matters here